app/proc/processor_test.go

```diff
@@ -0,0 +1,28 @@
+package proc
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetDefault(t *testing.T) {
+	p := Processor{
+		Conf: &Conf{},
+	}
+
+	assert.Equal(t, p.Conf.System.Concurrent, 0)
+	assert.Equal(t, p.Conf.System.MaxItems, 0)
+	assert.Equal(t, p.Conf.System.MaxTotal, 0)
+	assert.Equal(t, p.Conf.System.MaxKeepInDB, 0)
+	assert.Equal(t, p.Conf.System.UpdateInterval, time.Duration(0))
+
+	p.setDefaults()
+
+	assert.Equal(t, p.Conf.System.Concurrent, 8)
```

Comment:
I think this better should be replaced with a complete structure test instead of multiple individual tests.
This way in case somebody adds a new option and default for it, a test will fail which would be a reminder to write down new behavior by altering the test.

Reply:
Done